apollo-adminservice/src/main/java/com/ctrip/framework/apollo/adminservice/controller/ItemController.java

```diff
@@ -135,6 +138,21 @@ public void delete(@PathVariable("itemId") long itemId, @RequestParam String ope
     return BeanUtils.batchTransform(ItemDTO.class, itemService.findItemsWithOrdered(appId, clusterName, namespaceName));
   }
 
+  @GetMapping("/apps/{appId}/clusters/{clusterName}/namespaces/{namespaceName}/items/deleted")
+  public List<ItemDTO> findDeletedItems(@PathVariable("appId") String appId,
+                                        @PathVariable("clusterName") String clusterName,
+                                        @PathVariable("namespaceName") String namespaceName) {
+    List<Commit> commits = commitService.find(appId, clusterName, namespaceName, null);
+    if (Objects.nonNull(commits)) {
+      List<Item> deletedItems = commits.stream()
+          .map(item -> new ConfigChangeContentBuilder().convertJsonString(item.getChangeSets()).getDeleteItems())
```

Comment:
after making `ConfigChangeContentBuilder.convertJsonString` `static`, you don't need `new ConfigChangeContentBuilder()` to create so many instances